packages/wonder-blocks-dropdown/src/components/__tests__/multi-select.test.tsx

```diff
@@ -31,6 +31,19 @@ const doRender = (element: React.ReactElement) => {
     };
 };
 
+jest.mock("react-popper", () => ({
+    ...jest.requireActual("react-popper"),
+    Popper: jest.fn().mockImplementation(({children}) => {
+        // Mock `isReferenceHidden` to always return false (or true for testing visibility)
+        return children({
+            ref: jest.fn(),
+            style: {},
+            placement: "bottom",
+            isReferenceHidden: false, // Mocking isReferenceHidden
+        });
+    }),
+}));
+
```

Comment:
I followed TDD to confirm a fix for the bug where the selects would open using the down arrow key. The SingleSelect and MultiSelect tests had different behaviours. Turns out, SingleSelect had a mock for `react-popper`. I've copied it over to MultiSelect so they have the same configuration!